### PR TITLE
Restructure docs html to match https://segment.com/docs/ in structure

### LINKED ISSUE #9: mvp4 - Create a top navigation pane, similar to https://segment.com/docs/
https://github.com/Shippable/pm/issues/8293

### LINKED ISSUE #10: mvp4 - Add the left navigation pane with up to 2 levels. Headings in the left pane can be expanded or collapsed. Clicking on any subheading opens an html page.
https://github.com/Shippable/pm/issues/8293

### PR DESCRIPTION
**Pull request by 0x-x0**
image

##### footer
image

closes https://github.com/Shippable/docs/issues/9
closes https://github.com/Shippable/docs/issues/10

### CODE REVIEW

File: theme/saas/static/css/shippable.css [hunk 1]
```diff
@@ -142,3 +142,257 @@ blockquote {
...
+    color: #00386c;
+    opacity: 1
+}
+
+.ship-nav-item.active .ship-nav-item-nav:empty {
+    display: none
+}
+
+html {
+    font: 16px/27px Segment Circular Std,Helvetica Neue,Helvetica,Arial,sans-serif;
```
Comment (rageshkrishna) [inline theme/saas/static/css/shippable.css]: Segment Circular Std?

**Comment by rageshkrishna** [comment]
@chetantarale I'd like to see this working locally before deciding to merge. Let's take a look at this tomorrow morning.

**Comment by 0x-x0** [comment]
Ok. Sure

**Comment by manishas** [comment]
i see the left navigation pane but i don't see the top navigation pane yet...

**Comment by 0x-x0** [comment]
closing this as have raised another PR for this

**Pull request closed without merging**